docs/section5/Rule5-4.md

```diff
@@ -24,20 +24,20 @@
 
 - For each building segment in the Baseline model: ```for building_segment_baseline in B_RMR.building.building_segments:```  
 
-  - For each thermal_block from building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
+  - For each thermal_block in building segment: ```for thermal_block_baseline in building_segment_baseline.thermal_blocks:```  
 
-  - For each zone in thermal block: ```zone_baseline in thermal_block_baseline.zones:```  
+  - For each zone in thermal block: ```for zone_baseline in thermal_block_baseline.zones:```  
 
-  - For each space in thermal zone: ```space_baseline in zone_baseline.spaces:```  
+  - For each space in thermal zone: ```for space_baseline in zone_baseline.spaces:```  
 
     - Get space conditioning type: ```space_conditioning_type_baseline = space_baseline.conditioning_type```  
 
-      - Get baseline contruction from Table G3.4-1 to G3.4-8 based on space conditioning type, status type and function type: ```surface_performance_target = data_lookup(climate_zone,space_conditioning_type_baseline,"Walls, Above-Grade")```  
+      - Get baseline contruction from Table G3.4-1 to G3.4-8 based on climate zone, space conditioning type and function type: ```surface_performance_target = data_lookup(table_G3_4,climate_zone,space_conditioning_type_baseline,"Walls, Above-Grade")```  
 
     - For each surface in space: ```for surface_baseline in space_baseline.surfaces:```  
 
       - Get the surface construction if the surface is above-grade wall: ```if ( surface_baseline.classification == "WALL" ) AND ( surface_baseline.adjacent_to == "AMBIENT" ): surface_construction_baseline = surface_baseline.construction```  
 
       - Get the performance values for the construction: ```surface_performance_value_baseline = surface_construction_baseline.u_factor```  
 
-    **Rule Assertion:** Baseline below-grade wall consruction modeled matches Table G3.4-1 to G3.4-8: ```surface_performance_value_baseline == surface_performance_target```  
+    **Rule Assertion:** Baseline below-grade wall construction modeled matches Table G3.4-1 to G3.4-8: ```surface_performance_value_baseline == surface_performance_target```  
```

Comment:
Also need to confirm the layer-by-layer description if present in the construction data group based on Appendix A. "The steel stud framing is a minimum uncoated thickness of 0.043 in. for 18 gage or 0.054 in. for 16 gage. The U-factors include R-0.17 for exterior air film, R-0.08 for stucco, R-0.56 for 0.625 in. gypsum board on the exterior, R0.56
for 0.625 in. gypsum board on the interior, and R-0.68 for interior vertical surfaces air film. "